internal/commands/app/create.go

```diff
@@ -105,9 +105,15 @@ func (cmd *CommandCreate) Handler(profile *cli.Profile, ui terminal.UI, clients
 		return nil
 	}
 
+	newApp, err := clients.Realm.CreateApp(groupID, cmd.inputs.Name, realm.AppMeta{cmd.inputs.Location, cmd.inputs.DeploymentModel})
```

Comment:
driveby fix for https://jira.mongodb.org/browse/REALMC-8326 ... @nathan feel free to take this if we want saner commits